examples/go-mq-basic/config.go

```diff
@@ -0,0 +1,42 @@
+package mqbasicrouter
+
+import (
+	"fmt"
+	"log/slog"
+	"net/http"
+	"time"
+)
+
+// Config provides the information to create a MQRouter
+type Config struct {
+	Log             *slog.Logger
+	RouteServerAddr string // address of the routeserver
+	PullInterval    time.Duration
+	Baseline        *BaselineWorkload
+	SandboxName     string
```

Comment:
Can you add some comments for each of these fields and what they represent in the MQ context?

Also, I'm thinking `SandboxName` is too vague here, and the `""` meaning implicitly that it's not a sandboxed workload makes the example less clear - maybe we could consider a more compound structure here:

```
Sandbox {
    isSandboxed: true | false
    name: "..."
}
```

Open to other suggestions so that the config params are more self-explanatory.

Reply:
If this is clunky, another alternative is to remove the parameter and have it be entirely inferred from the env var, noting that the presence of the env var indicates that it'll behave as if it's in a sandbox.

Reply:
I updated both examples.
WDYT now?

Reply:
Looks great! Thanks for addressing the comments